README.md

```diff
@@ -21,6 +21,19 @@ To bind the java agent to a specific IP change the port number to `host:port`.
 
 See `./run_sample_httpserver.sh` for a sample script that runs the httpserver against itself.
 
+### Wildfly
```

Comment:
I don't think we should be mentioning particular pieces of software that are doing non-standard things, particularly not above the primary reference docs.

These docs belong on Wildfly's end.